cli/tests/data/help/task.txt

```diff
@@ -4,11 +4,20 @@ Description:
 Usage:
     dcos task --help
     dcos task --info
+    dcos task attach [--interactive --pty] [<task>]
+    dcos task exec [--interactive --pty] [<task>] [<cmd>]
```

Comment:
Should we drop the `attach` for now as well the `--pty` (which should be `-tty` in the long run).